site/pages/disclaimer.tsx

```diff
@@ -0,0 +1,17 @@
+import { GetStaticProps } from "next";
+import { Disclaimer } from "components/pages/Disclaimer";
+import { loadTranslation } from "lib/i18n";
+
+type Props = Record<string, unknown>;
+
+export const getStaticProps: GetStaticProps<Props> = async (ctx) => {
+  const translation = await loadTranslation(ctx.locale);
+  return {
+    props: {
+      translation,
+    },
+    revalidate: 60,
```

Comment:
no need for a revalidate on this since its static content. ill fix after i merge